sechub-developertools/scripts/pds-api.sh

```diff
@@ -47,6 +47,11 @@ export PDS_APITOKEN="pds-apitoken"
 EOF
 }
 
+function curl_with_pds_auth {
+  # Don't reveal secrets in the curl process
+  printf "user = $PDS_USERID:$PDS_APITOKEN\n" | curl -K - $CURL_PARAMS "$@"
```

Comment:
`-K` is not especially intuitive.

```suggestion
  printf "user = $PDS_USERID:$PDS_APITOKEN\n" | curl --config - $CURL_PARAMS "$@"
```

Reply:
implemented